Docs/button.html

```diff
@@ -4,9 +4,286 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Button | Docs</title>
+    <title>Buttons | AceUI</title>
+
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="../Styles/docs.css" />
+    <link rel="stylesheet" href="../Styles/Components/button.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
   </head>
-  <body>
-    <h1>BUTTON DOCS</h1>
+  <body class="flex-column">
+    <!------ Main Header ------>
+    <header class="main-header flex-row">
+      <div class="main-header-logo mx-2 my-05">
+        <a href="#">
+          <h2 class="text-bolder">ace.</h2>
+        </a>
+      </div>
+      <nav class="main-nav mx-2">
+        <ul class="main-nav-links">
+          <li class="main-nav-link px-1">
+            <h6 class="text-bold">
+              <a href="./Docs/button.html"
+                >Docs<i class="fa fa-file px-05" style="font-size: 20px;"></i
+              ></a>
+            </h6>
+          </li>
+          <li class="main-nav-link px-1">
+            <h6 class="text-bold">
+              <a href="https://github.com/Ayesha-7/Component-Library"
+                >Github<i
+                  class="fa fa-github px-05"
+                  style="font-size: 24px;"
+                ></i>
```

Comment:
here also dont use inline styles

Reply:
thanks for pointing out Richa, removed all inline styles.